subliminal/providers/addic7ed.py

```diff
@@ -240,7 +240,7 @@ def query(self, series, season, year=None, country=None):
         # get the show id
         show_id = self.get_show_id(series, year, country)
         if show_id is None:
-            logger.error('No show id found for %r (%r)', series, {'year': year, 'country': country})
+            logger.warning('No show id found for %r (%r)', series, {'year': year, 'country': country})
```

Comment:
In the context of the "query" function, I believe this is an error, not a warning.